package.json

```diff
@@ -143,7 +143,7 @@
     "jade": "1.11.0",
     "jade-loader": "0.8.0",
     "joi": "10.4.1",
-    "jquery": "3.2.1",
+    "jquery": "3.3.1",
```

Comment:
Can you change this to `^3.3.1`? (We're gradually moving towards specifying versions using `^`)

And it looks like yarn is now resolving to two jquery deps. To "give yarn the right kick", maybe you could edit the `yarn.lock` file and remove the lines for `jquery@^3.2.1:` and re-run `yarn`? Hopefully that should make it so yarn resolves to a single jquery version 

(it's trying to be conservative with the changes, but here we want to make sure it resolves to a single dep)

Reply:
@kjbekkelund it seems https://github.com/elastic/eui/blob/master/package.json#L30 requires `^3.2.1`, and that (possibly) causes yarn to add them both.  Removing 3.2.1 and running `yarn` brings it right back. This is very strange, because the `^` implies that it should use anything up to 4.0 version.

Reply:
Yeah, you might just need to jump into `yarn.lock` file directly and make the change. I pulled this PR locally and jumped into `yarn.lock` and removed this:

```diff
-jquery@^3.2.1:
-  version "3.2.1"
-  resolved "https://registry.yarnpkg.com/jquery/-/jquery-3.2.1.tgz#5c4d9de652af6cd0a770154a631bba12b015c787"
-
```

then I ran `yarn` again. After that my full diff is:

```diff
-jquery@3.3.1:
+jquery@3.3.1, jquery@^3.2.1:
   version "3.3.1"
   resolved "https://registry.yarnpkg.com/jquery/-/jquery-3.3.1.tgz#958ce29e81c9790f31be7792df5d4d95fc57fbca"

-jquery@^3.2.1:
-  version "3.2.1"
-  resolved "https://registry.yarnpkg.com/jquery/-/jquery-3.2.1.tgz#5c4d9de652af6cd0a770154a631bba12b015c787"
-
```

Which means yarn "got the right kick" when I removed the current resolution it had.

(There might be better ways of doing this — it's still a learning process for me)

Just let me know if you want to jump on zoom to go through this. I'm heading to bed now, but I'll be available tomorrow.

Reply:
(just to be clear: my final diff shows both versions resolving to the same version — `version "3.3.1"`, which means we'll only have one version of jQuery present)

(also, sidenote, we're working on some tooling to make sure the build fails when we've got multiple versions of a dep, plus some good explanations for debugging/handling it)

Reply:
@kjbekkelund very strange, I did the same thing, and for some reason it re-added jquery 3.2.1 back to the lock file.  But now I did it one more time, and it worked. Magic.  Thanks for looking into it!  Resubmitted.